Display review data

ghickman opened the pull request:
Add per-Release and per-ReleaseFile metadata

**Release**
We're just saving raw data to the metadata for Release currently so this isn't the most useful but we can at least show that we've captured parts of it?
![](https://p198.p4.n0.cdn.getcloudapp.com/items/4gueYlmP/09af33cb-0094-436b-be97-c079c0846b61.jpg?v=15d6d900f2b55dbaf681a933ffbc10f0)

**ReleaseFile**
![](https://p198.p4.n0.cdn.getcloudapp.com/items/wbuQAl2j/2dc2d262-ed2f-404b-89b1-e1c9cc52c7b7.jpg?v=cc5fddf79bcd17a83a699157b9730bd4)

bloodearnest commented on assets/src/scripts/outputs-viewer/components/Metadata/Metadata.jsx:
```diff
@@ -46,6 +53,19 @@ function Metadata({ fileDate, fileName, fileSize, fileUrl }) {
       </ul>
+      <div>
+        <div className="mb-3">
+          <strong>What does this file contain?</strong>
```
This should be the key value of the dict

bloodearnest commented on assets/src/scripts/outputs-viewer/components/Metadata/Metadata.jsx:
```diff
@@ -46,6 +53,19 @@ function Metadata({ fileDate, fileName, fileSize, fileUrl }) {
+
+        <div>
+          <strong>
+            How has the author ensured that this file is non-disclosive?
```
Same as above.

bloodearnest commented on assets/src/scripts/outputs-viewer/App.jsx:
```diff
@@ -137,6 +137,8 @@ function App({
               {isReviewEdit ? <FileReview selectedFile={selectedFile} /> : null}
               <Card>
                 <Metadata
+                  context={selectedFile.metadata.context}
```
This should be just `metadata=metadata`

The jsx component should iterate through the key/value pairs.

To match the current document, we'd expect a dict like:

````
{
    "Context": "user input context",
    "Controls": "user input controls",
}
```

ghickman commented:
Updated to show what it looks like.  I'm not sold on using the keys for the labels as that ties the display to the labels of the form, but I've updated it to that so we can see what it looks like for the demo.

bloodearnest commented:
The keys have no semantic meaning - the can be the whole text of the form label if they want.

In fact, a dict of labels: values is a much better terminology
